docs/dev/lsp-extensions.md

```diff
@@ -372,7 +372,7 @@ interface Runnable {
 }
 ```
 
-rust-analyzer supports only one `kind`, `"cargo"`. The `args` for `"cargo"` look like this:
+rust-analyzer supports two `kind`s of runnables, `"cargo"` and `"rustproject"`. The `args` for `"cargo"` look like this:
```

Comment:
`projectjson`, not `rustproject` right?

Reply:
Yes, sorry! I get the two switched up in my head.